pycaret/datasets.py

```diff
@@ -3,7 +3,7 @@
 # License: MIT
 
 
-def get_data(dataset="index", save_copy=False, profile=False, verbose=True):
+def get_data(dataset="index", save_copy=False, profile=False, verbose=True,address = "https://raw.githubusercontent.com/pycaret/pycaret/master/datasets/"):
```

Comment:
Can this be formatted with black? thanks

Reply:
Sure.